### ISSUE TITLE
Monika can't read my gifts?

### ISSUE THREAD

**Issue by Mcmanybucks** [Monika-After-Story/MonikaModDev#4682]
I'm trying to give her shellbikini.gift,  but she can't read it.. I put it into /characters.

**Comment by multimokia** [comment]
Did you install the spritepack?

**Comment by Mcmanybucks** [comment]
> 
> 
> Did you install the spritepack?

I don't know what that is so I'll go with no... I just updated the game though.

**Comment by multimokia** [comment]
Then you don't have the shellbikini as a part of your game. These outfits are to be installed separately. You'll want to see [this link](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#installing-sprite-packs) for installation instructions (also where to get them)

**Comment by Mcmanybucks** [comment]
> 
> 
> Then you don't have the shellbikini as a part of your game. These outfits are to be installed separately. You'll want to see [this link](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#installing-sprite-packs) for installation instructions (also where to get them)

Ok so I've installed them now, and I've added the shellbikini.gift in the character folder again.. she doesn't notice it.

**Comment by multimokia** [comment]
Yeah, you'll have to wait for tomorrow

**Comment by Rin3105** [comment]
I have already asked the same question as you in this issue, try to follow exactly what I did in there:
https://github.com/Monika-After-Story/MonikaModDev/issues/4664

**Comment by ThePotatoGuy** [comment]
[closing as resolved]

**Comment by Loos6789** [comment]
hi there! in case anyone from this time up, I just wanna give a pointer to how I fixed mine and got it working. so when you put the gift into the characters file, click view. Once there press show/hide (if you don't have that option it may not matter). one done, click the option that says "file name extensions". you will then have a little tick on the box. Then your done! idk why it worked or why it DOES work, but that's how I got mine too. hope it helped
